.github/workflows/main.yml

```
@@ -0,0 +1,27 @@
+on: [push]
+
+jobs:
+  clippy:
```

clippy というよりは lint という job ですね

```suggestion
  lint:
```